app/build.gradle

```diff
@@ -88,7 +88,7 @@ dependencies {
     kapt "com.github.Raizlabs.DBFlow:dbflow-processor:$rootProject.dbflowVersion"
     implementation "com.github.Raizlabs.DBFlow:dbflow-core:$rootProject.dbflowVersion"
     implementation "com.github.Raizlabs.DBFlow:dbflow:$rootProject.dbflowVersion"
-
+    implementation 'io.michaelrocks:libphonenumber-android:8.10.16'
```

Comment:
Is this dependency related to your changes in this PR? If yes, then please move the dependency version to `ext` block